docs/getting-started/setup/optimizing-build-size.md

```diff
@@ -0,0 +1,435 @@
++ import { Table } from '@ckeditor/ckeditor5-table/dist/index.js';
+```
+
+<info-box warning>
+	Note the `/dist/index.js` part of the import paths. This is important to ensure that the editor functions are imported from the correct files.
+</info-box>
+
+To find the correct package names, see the {@link getting-started/legacy-getting-started/legacy-imports#finding-individual-packages Finding individual packages} guide. Alternatively, if you are using an IDE with TypeScript support, you can use the `Go to Definition` feature to find the package names.
+
+### Styles
+
+Currently, you probably import one or both of the following style sheets, depending on whether or not you use the premium features:
+
+```js
+import 'ckeditor5/ckeditor5.css';
+import 'ckeditor5-premium-features/ckeditor5-premium-features.css';
+```
+
+Such imports are great, because they are very readable and easy to follow. However, these style sheets contain all the styles for all the plugins and features. If you want to reduce the build size, you can import only the core styles and the styles for the plugins that you use.
+
+First, import the core styles. They are all needed for the editor to work properly:
+
+```js
+// Import the core styles.
+import '@ckeditor/ckeditor5-theme-lark/dist/index.css';
+import '@ckeditor/ckeditor5-clipboard/dist/index.css';
+import '@ckeditor/ckeditor5-core/dist/index.css';
+import '@ckeditor/ckeditor5-engine/dist/index.css';
+import '@ckeditor/ckeditor5-enter/dist/index.css';
+import '@ckeditor/ckeditor5-paragraph/dist/index.css';
+import '@ckeditor/ckeditor5-select-all/dist/index.css';
+import '@ckeditor/ckeditor5-typing/dist/index.css';
+import '@ckeditor/ckeditor5-ui/dist/index.css';
+import '@ckeditor/ckeditor5-undo/dist/index.css';
+import '@ckeditor/ckeditor5-upload/dist/index.css';
+import '@ckeditor/ckeditor5-utils/dist/index.css';
+import '@ckeditor/ckeditor5-watchdog/dist/index.css';
+import '@ckeditor/ckeditor5-widget/dist/index.css';
+```
+
+Then, import the styles for the plugins that you use. For example, if you use the `Bold`, `Italic` or `Table` features, you can import only the styles for those features:
+
+```js
+// Import the styles for the features that you use.
+import '@ckeditor/ckeditor5-basic-styles/dist/index.css';
+import '@ckeditor/ckeditor5-table/dist/index.css';
+// ...
+```
+
+When it comes to plugin styles, the rule of thumb is to import the styles from all the individual packages from which you import the editor features. For example, if you import the `Bold` feature from `@ckeditor/ckeditor5-basic-styles/dist/index.js`, you should also import `@ckeditor/ckeditor5-basic-styles/dist/index.css`.
+
+You may notice that some plugin style sheets are empty. This is intentional, as some plugins do not have styles now, but may have them in the future. Adding the imports now will ensure that you do not accidentally miss some styles if that happens. Importing empty stylesheets does not increase the build size.
+
```

Comment:
Maybe would be good to also mention somewhere here about CSS that not only `index.css` can be imported but it's divided into `content` and `editor`? Sounds also like a potential optimization, WDYT?

Reply:
I was thinking the same thing, but it's already explained in a separate document and I don't want to duplicate it.

https://ckeditor5.github.io/docs/nightly/ckeditor5/latest/getting-started/setup/css.html

Reply:
Yes, thats true.

Maybe a link with short sentence to it could be a good idea? :thinking: 

Its always better to have it in one place (i mean all the necessary links that could help) - when someone will enter this article right from the search engine (like google) will reach all necessary information.

Reply:
Maybe just link it, then?

Reply:
Done :heavy_check_mark: